cli/txctx/tx.go

```diff
@@ -69,6 +74,18 @@ func SignAndSend(ctx *cli.Context, act *actor.Actor, acc *wallet.Account, tx *tr
 			tx.ValidUntilBlock += uint32((waitTime.Milliseconds() / int64(ver.Protocol.MillisecondsPerBlock))) + 1
 		}
 		_, _, err = act.SignAndSend(tx)
+
+		if ctx.Bool("await") {
+			aer, err := act.Wait(tx.Hash(), tx.ValidUntilBlock, nil)
+			if err != nil {
+				return cli.NewExitError(fmt.Errorf("failed to await transaction: %w", err), 1)
+			}
+			if aer.FaultException != "" {
+				return cli.NewExitError(fmt.Errorf("transaction failed: %s", aer.FaultException), 1)
+			}
+			fmt.Fprintln(ctx.App.Writer, "transaction is accepted to the block, transaction hash: "+tx.Hash().StringLE())
+			return nil
+		}
```

Comment:
@AnnaShaleva  do we really need the blockhash and VM state info?  because with this information we can't use e.CheckTxPersisted(t, "transaction is accepted to the block, transaction hash: "), so should we create a similar CheckTxPersisted for taking into account blockhash and VM state?

Reply:
Yes, create it for tests. Some `CheckAwaitableTxPersisted` will suit. Move the common part of `CheckTxPersisted to a separate unexported method and reuse it from `CheckTxPersisted` and from `CheckAwitableTxPersisted`.